### PR TITLE
ConOps Draft

### PR DESCRIPTION
**Pull request by KenwoodFox**
This PR/branch is for the ConOps section.

it should be merged when the ConOps are ready to present!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +50,5 @@ Look to...
 
 Start by cloning this repo down and making a new branch, offical team communication
 is through discord so hop on over any time! If you're new to git check out this video: https://youtu.be/mJ-qvsxPHpY
+
+Hi Joe
```
Comment (KenwoodFox) [inline README.md]: Im gonna revert this but thank you :)

**Comment by KenwoodFox** [comment]
If anybody is havind trouble building with kile, here's the [ConOps.pdf](https://github.com/KenwoodFox/EG-310-InvertedPendulum/files/10354110/ConOps.pdf) file.

**Review by KenwoodFox — COMMENTED** [review]
Review the conops changes by going to here and clicking review changes! https://github.com/KenwoodFox/EG-310-InvertedPendulum/pull/2/files

image

**Comment by KenwoodFox** [comment]
We'll review this later today w prof

**Review by Matthew-Handley — APPROVED** [review]
:D looks good

**Pull request merged**